### ISSUE TITLE
Static Egress Gateway

### ISSUE THREAD

**Issue by chasewilson** [Azure/AKS#4638]
Create a Static Egress Gateway Nodepool and assign an IP prefix to is. Annotate pods to send traffic via Static Egress Gateway rather than outbound type.

**Comment by illrill** [comment]
Hey @chasewilson,

Will you be recommending current users of NAT Gateway to plan for a migration to Egress Gateway? What are the added benefits?

**Comment by chasewilson** [comment]
@illrill, no. Static Egress Gateway's intent is to provide a way to granularly controll access to resources outside the cluster at the pod level. Not as a replacement for NAT Gateway. Statigc Egress Gateway isn't an outbound type and clusters will need to still have an outbound type defined and can annotate pods/deployments to use Static egress as an addition to thier outbound type.
